.travis.yml

```diff
@@ -9,12 +9,8 @@ dist: xenial
 language: rust
 
 install:
-  - rustup component add clippy rustfmt
-  - sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
-  - $(aws ecr get-login --no-include-email --region $AWS_REGION)
```

Comment:
Do we still need to remove the node-gyp directory?

Reply:
Seemed like a one time thing that fixed itself after I deleted once and ran that through travis.

Reply:
Still needed apparently. I want to investigate that in a separate PR.